src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1029,36 +1034,47 @@ export default class CanvasController {
     }
 
     /**
+     * Draw the RADAR RETURN AND HISTORY DOTS ONLY of all radar target models
+     *
      * @for CanvasController
-     * @method canvas_draw_all_aircraft
+     * @method canvas_draw_radar_targets
      * @param cc
      */
-    canvas_draw_all_aircraft(cc) {
-        for (let i = 0; i < prop.aircraft.list.length; i++) {
+    canvas_draw_radar_targets(cc) {
+        const radarTargetModels = this._scopeModel.radarTargetCollection.items;
+
+        for (let i = 0; i < radarTargetModels.length; i++) {
             cc.save();
-            this.canvas_draw_aircraft(cc, prop.aircraft.list[i]);
+            this.canvas_draw_radar_target(cc, radarTargetModels[i]);
             cc.restore();
         }
+        // for (let i = 0; i < prop.aircraft.list.length; i++) {
```

Comment:
remove this commented out block if it is deprecated

Reply:
Done